object.c

```diff
@@ -4392,118 +4385,117 @@ InitVM_Object(void)
-     * - #pp:: Prints the given objects in pretty form.
-     * - #print:: Prints the given objects to standard output without a newline.
-     * - #printf:: Prints the string resulting from applying the given format string
-     *             to any additional arguments.
-     * - #putc:: Equivalent to <tt.$stdout.putc(object)</tt> for the given object.
-     * - #puts:: Equivalent to <tt>$stdout.puts(*objects)</tt> for the given objects.
-     * - #readline:: Similar to #gets, but raises an exception at the end of file.
-     * - #readlines:: Returns an array of the remaining lines from the current input.
-     * - #select:: Same as IO.select.
+     * - ::pp: Prints the given objects in pretty form.
+     * - #gets: Returns and assigns to <tt>$_</tt> the next line from the current input.
+     * - #open: Creates an IO object connected to the given stream, file, or subprocess.
+     * - #p:  Prints the given objects' inspect output to the standard output.
+     * - #print: Prints the given objects to standard output without a newline.
+     * - #printf: Prints the string resulting from applying the given format string
+     *   to any additional arguments.
+     * - #putc: Equivalent to <tt.$stdout.putc(object)</tt> for the given object.
+     * - #puts: Equivalent to <tt>$stdout.puts(*objects)</tt> for the given objects.
+     * - #readline: Similar to #gets, but raises an exception at the end of file.
+     * - #readlines: Returns an array of the remaining lines from the current input.
+     * - #select: Same as IO.select.
      *
      * === Procs
      *
-     * - #lambda:: Returns a lambda proc for the given block.
-     * - #proc:: Returns a new Proc; equivalent to Proc.new.
+     * - #lambda: Returns a lambda proc for the given block.
+     * - #proc: Returns a new Proc; equivalent to Proc.new.
      *
      * === Tracing
      *
-     * - #set_trace_func:: Sets the given proc as the handler for tracing,
-     *                     or disables tracing if given +nil+.
-     * - #trace_var:: Starts tracing assignments to the given global variable.
-     * - #untrace_var:: Disables tracing of assignments to the given global variable.
+     * - #set_trace_func: Sets the given proc as the handler for tracing,
+     *   or disables tracing if given +nil+.
+     * - #trace_var: Starts tracing assignments to the given global variable.
+     * - #untrace_var: Disables tracing of assignments to the given global variable.
      *
      * === Subprocesses
      *
-     * - #`command`:: Returns the standard output of running +command+ in a subshell.
-     * - #exec:: Replaces current process with a new process.
-     * - #fork:: Forks the current process into two processes.
-     * - #spawn:: Executes the given command and returns its pid without waiting
-     *            for completion.
-     * - #system:: Executes the given command in a subshell.
+     * - {`command`}[#method-i-60]: Returns the standard output of running +command+ in a subshell.
```

Comment:
I think the canonical notation is `` #` `` here, and `` command` `` is extraneous.
Do you think `` #`command` `` should be parsed as single method reference?

Reply:
I'm agnostic on this one.  In the What's Here section, <tt>#`</tt> in source would become a link with just the one character <tt>`</tt>, a tiny target even if correct.  How would you like this to be?

Also, <tt>#`</tt> did not link on my machine (versions same as above).